src/routes/(app)/order/[id]/+page.svelte

```diff
@@ -294,6 +296,13 @@
 									/></a
 								>
 							{/if}
+							{#if payment.method === 'card'}
```

Comment:
```suggestion
							{#if payment.method === 'card' && !data.hideCbQrCode}
```

no need to render it in the DOM at all